### ISSUE TITLE
Build error: scalapack

### ISSUE THREAD

**Issue by mkrompiec** [GlobalArrays/ga#239]
Trying to compile in a conda environment, almost identical to https://github.com/conda-forge/nwchem-feedstock/blob/master/recipe/build.sh
l```
ibtool: compile:  mpif90 -DHAVE_CONFIG_H -I. -Ima -I./ma -I./LinAlg/lapack+blas -Iglobal/src -I./global/src -I./global/testing -I./pario/dra -I./pario/eaf -I./pario/elio -I./pario/sf -Igaf2c -I./gaf2c -I./tcgmsg -I./tcgmsg/tcgmsg-mpi -I./comex/src-armci -DNDEBUG -D_FORTIFY_SOURCE=2 -O2 -isystem /home/azureuser/anaconda3/envs/clean/include -fdefault-integer-8 -fno-aggressive-loop-optimizations -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -isystem /home/azureuser/anaconda3/envs/clean/include -fallow-argument-mismatch -c global/src/scalapack.F -o global/src/scalapack.o
global/src/scalapack.F:2454:15:

 2454 |       if (.not.ga_deallocate(g_b))
      |               1
Error: Function 'ga_deallocate' at (1) has no IMPLICIT type; did you mean 'ga_allocate'?
global/src/scalapack.F:3125:19:

 3125 |            if(.not.ga_deallocate(g_b))
      |                   1

```

**Comment by FreemanTheMaverick** [comment]
https://github.com/GlobalArrays/ga/issues/238#issuecomment-993811733
A similar problem.

**Comment by mkrompiec** [comment]
Indeed, this is a similar (the same?) problem, but with a different toolchain. I used GCC 11.2.0.

**Comment by edoapra** [comment]
The tarball has been updated with fix https://github.com/GlobalArrays/ga/commit/a84cb63584204a22efa806e9e1c2ea10688d57bc
